server/oauth2.go

```diff
@@ -246,7 +247,7 @@ type idTokenClaims struct {
 	Name string `json:"name,omitempty"`
 }
 
-func (s *Server) newIDToken(clientID string, claims storage.Claims, scopes []string, nonce, accessToken string) (idToken string, expiry time.Time, err error) {
+func (s *Server) newIDToken(clientID string, claims storage.Claims, scopes []string, nonce, accessToken string, connID string) (idToken string, expiry time.Time, err error) {
```

Comment:
nit: the `accessToken string,` is redundant because `connID` is a string. You can just do `accessToken, connID string`